pymssqlutils/databaseresult.py

```diff
@@ -40,6 +41,8 @@ def _get_data_mapper(sql_type_hint: int, item: Any) -> Callable[[Any], SQLParame
     if sql_type_hint == 2:  # BINARY: bytes, datetime, time, date
         if isinstance(item, (datetime, date, time)):
             return identity
+        if isinstance(item, uuid.UUID):
+            return str(item)
```

Comment:
my bad, this should just be `return str` (look at type hint above it should return a callable)